### ISSUE TITLE
Using CLI to compile whole directory also copies non-JS files over

### ISSUE THREAD

**Issue by callumlocke** [babel/babel#888]
```sh
$ babel --source-maps --out-dir=lib src
```

I was surprised to find that if there are extra, non-JavaScript files in the `src` directory (and subdirectories), these get copied over to `lib` as part of the compilation. (For comparison, the equivalent command in CoffeeScript only compiles `.coffee` files and ignores any other file types.)

Is there a flag to disable this copying? (Or is it a bug?)

**Comment by sebmck** [comment]
This is expected behaviour. No there's currently no way to disable it. What files are there being copied that you don't want?

**Comment by callumlocke** [comment]
I've got a `src` directory full of `.coffee` files that I'm gradually porting to ES6 `.js` files, one by one. This is a big job that needs to be done one file at a time, testing all the way. So I have a mixture of files in my `src` dir. My build command is something like this:

```sh
$ babel --source-maps --out-dir lib src && coffee --map --output lib --compile src
```

This unfortunately copies over all my `.coffee` files into `lib`.

What is this feature for? When would people want Babel to copy over random non-JS stuff that it happens to find in the source directory? It seems a bit over-helpful :)

Babel is awesome btw, I love it

**Comment by sebmck** [comment]
> What is this feature for? When would people want Babel to copy over random non-JS stuff that it happens to find in the source directory? It seems a bit over-helpful :)

Yeah, I originally added it because I was working on making Babel self-hosting and there are some non-`.js` files and I wanted them copied over. Should be able to safely remove it since I'm not likely going to get around to self-hosting anytime soon and if I do it wont be via the CLI.

> Babel is awesome btw, I love it

Thanks!

**Comment by callumlocke** [comment]
:+1: that was quick

**Comment by sebmck** [comment]
Released as of 4.5.0, thanks!
